ovirt/resource_ovirt_vm.go

```diff
@@ -636,6 +684,18 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	// update the VM with the auto pinning
+	if isAutoPinning {
```

Comment:
Why that part can't be joined with the above logic? before the VM is built?

Reply:
Because we will hit the feature constrains. We can do it upon creation **only** if no CPU topology provided to the VM.
The way we do it now (since engine 4.4.5), will split the calls and after the VM created we no longer have the conflict.